src/elvis_style.erl

```diff
@@ -304,9 +305,23 @@ result_node_line_col_fun(Msg) ->
 
 %% Rule checking
 
+-spec line_is_comment(binary()) -> true | false.
+line_is_comment(Line) ->
+    case re:run(Line, "^ *%") of
```

Comment:
I think you should use `^\s*` or `^[ \t]*` instead of `^ *` to account for lines that start with tabs.
And also you're only skipping those lines that are whole comments. I assume it's on purpose, right? e.g. the following line will result in a warning even if options are `#{limit => 60, skip_comments => true}`.

```erlang
  Timeout = 7 * 24 * 60 * 60 * 1000, % A week in milliseconds
```